Release.toml

```diff
@@ -30,3 +30,6 @@ version = "1.0.7"
     "migrate_v1.0.6_admin-container-v0-6-0.lz4",
 ]
 "(1.0.6, 1.0.7)" = []
+"(1.0.7, 1.0.8)" = [
+    "migrate_v1.0.8_control-container-v0-5-0.lz4",
```

Comment:
#1443 is also affecting this list, and I'd rather not leave it up to a last-minute merge, so I'd like to get one done and rebase the other; it seems like admin container is closer, so I'm leaving this comment here as a reminder :)

(same in sources/Cargo.toml)